.lando.yml

```diff
@@ -9,48 +9,21 @@ config:
   database: mariadb
   xdebug: false
 
+env_file:
+  - config/.env
+  - .env
+
 services:
   appserver:
     composer:
       # Note installing phpunit should be temporary because core should really install it via composer on its own.
       phpunit/phpunit: "^6"
     install_dependencies_as_root:
-      - apt-get update -y
-      - |
-        apt-get -y install libyaml-dev;
-        yes | pecl install yaml;
-        echo 'extension=yaml.so' > /usr/local/etc/php/conf.d/yaml.ini
-      - apt-get install gnupg -y
-      - |
-        curl -sL https://deb.nodesource.com/setup_10.x | bash -;
-        apt-get install nodejs -y
-      - |
-        curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -;
-        echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list;
-        apt-get update && apt-get install yarn -y
-      - apt-get install zip -y
-      - apt-get install subversion -y
+      - bash bin/install-dependencies.sh
       # Allow loopback requests to work.
       - echo "127.0.0.1  wordpressdev.lando.site" >> /etc/hosts
-    install_dependencies_as_me:
-      # Setup environment variables and WP-CLI configuration.
-      - if [ ! -e $LANDO_MOUNT/.env ]; then cp $LANDO_MOUNT/config/.env $LANDO_MOUNT/.env && sed -i "s#/app#$LANDO_MOUNT#" $LANDO_MOUNT/.env; fi
-      - if [ ! -e $LANDO_MOUNT/wp-cli.yml ]; then cp $LANDO_MOUNT/config/wp-cli.yml $LANDO_MOUNT/wp-cli.yml; fi
```

Comment:
I don't think we can ditch this. Maybe the `.env` file, since we could tell Lando to use it from `config/.env` - but the `wp-cli.yml` file should still be copied over there if it doesn't exist.

Reply:
It's not ditched. It's done in `bin/setup-wordpress.sh` now instead:

https://github.com/felixarntz/wordpressdev/blob/6ecc2c969bdf6ee25dfc029001ff6a7a24f628a9/bin/setup-wordpress.sh#L13-L15

Reply:
Ah I missed this. 👍